packages/io/svg-serializer/index.js

```diff
@@ -203,7 +203,7 @@ const convertPath = (path, offsets, options) => {
   return str
 }
 
-const convertColor = (color) => `rgb(${color[0] * 255},${color[1] * 255},${color[2] * 255},${color[3] * 255})`
```

Comment:
I never noticed this before... but should 'rgb' become 'rgba'? I believe this should follow the specifications.

https://www.w3.org/TR/css-color-3/#rgba-color

Reply:
@btakita 

I just checked all browsers using rgba(r,g,b,a). All browsers rendered properly.

Please change rgb to rgba

Reply:
Done